templates/ipsec.conf

```diff
@@ -21,7 +21,8 @@ conn vpnpsk
   type=transport
   ike=3des-sha1,aes-sha1
   ikev2=no
-  phase2alg=3des-sha1,aes-sha1
+  phase2alg=3des-sha1,aes-sha1,aes256-sha2
```

Comment:
Was this change required? @ttheune said he wasn't sure it was necessary.

Reply:
This is the error being displayed.
https://support.apple.com/en-us/HT211840

Reply:
Will be going with an expanded phase2alg list with the xauth build soon, so this is okay either way.